modules/initialLoad.js

```diff
@@ -0,0 +1,26 @@
+import grabber from './grabber.js';
+import Book from './book.js';
+
+const template = document.createElement('template');
+const bookf = new Book();
+
+export default function initialLoad() {
+  const oldBooks = JSON.parse(localStorage.getItem('booksArray')) || [];
```

Comment:
- [ ] Same here. I would recommend using the arrow syntax for the function. As it is requirement to use ES6 syntax as much as you can to get familiar with.